{
  "Header": {
    "Original": "CVE-2018-10906 (In fuse before versions 2.9.8 and 3.x before 3.2.5, fusermount is vuln ...)",
    "Line": 344498,
    "ID": "CVE-2018-10906",
    "Description": "(In fuse before versions 2.9.8 and 3.x before 3.2.5, fusermount is vuln ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-4257-1 DLA-1468-1}",
      "Line": 344499,
      "Type": "xref",
      "Bugs": [
        "DSA-4257-1",
        "DLA-1468-1"
      ]
    },
    {
      "Original": "- fuse3 3.2.6-1 (bug #911343)",
      "Line": 344500,
      "Type": "package",
      "Package": "fuse3",
      "Kind": "fixed",
      "Version": "3.2.6-1",
      "BugNo": 911343
    },
    {
      "Original": "- fuse 2.9.8-1 (bug #904439)",
      "Line": 344501,
      "Type": "package",
      "Package": "fuse",
      "Kind": "fixed",
      "Version": "2.9.8-1",
      "BugNo": 904439
    },
    {
      "Original": "NOTE: https://github.com/libfuse/libfuse/pull/268",
      "Line": 344502,
      "Type": "NOTE",
      "Description": "https://github.com/libfuse/libfuse/pull/268"
    },
    {
      "Original": "NOTE: https://sourceforge.net/p/fuse/mailman/message/36374753/",
      "Line": 344503,
      "Type": "NOTE",
      "Description": "https://sourceforge.net/p/fuse/mailman/message/36374753/"
    }
  ]
}